packages/victory-core/package.json

```diff
@@ -47,7 +52,8 @@
     "build": {
       "dependencies": [
         "build:lib",
-        "build:dist"
+        "build:dist",
+        "types:create"
```

Comment:
Our other tasks depend on things at the level of “build:lib:esm”, etc.

Do any of our other tasks need types created or is it solely for publishing?

Reply:
Come to think of it, though, our `lint` task also depends on types, so I'll be sure to add it there too.

Reply:
Sounds good